base_multi_company/models/res_company_assignment.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class ResCompanyAssignment(models.Model):
+
```

Comment:
@lasley It would be nice to describe the meaning of this special model: the motivation, how it works and where the view is created. Indeed it will be hard for a developer how has not followed the discussion in this PR to understand why this model exists etc....